website/pages/docs/deployment/ecs.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Running CloudQuery on Amazon ECS
+tag: tutorial
+date: 2023/03/03
+---
+
+# Running CloudQuery on Amazon ECS
+
+In this tutorial we will be deploying CloudQuery on AWS ECS using Fargate. You will be using the AWS CLI to create the required resources. You can also use the AWS Console to create the resources. At the end of the tutorial you will have a CloudQuery instance running on AWS ECS that will collect data from your AWS account and store it in an S3 bucket. You can then query the data using Athena.
+
+## Prerequisites
+Before starting the deployment process, you need to have the following prerequisites:
+  * An AWS account
```

Comment:
```suggestion
```

Isn't this included in the AWS CLI prerequisite?

Also can we add link to the relevant AWS doc for each prerequisite?